package.json

```diff
@@ -116,7 +116,8 @@
     "classlist.js": "1.1.20150312",
     "core-js": "3.7.0",
     "focus-trap": "6.5.0",
-    "fundamental-styles": "0.19.0-rc.38",
+    "fundamental-styles": "0.19.0-rc.42",
+    "hammerjs": "2.0.8",
```

Comment:
Why do you bring `hammerjs`? @N1XUS recently created a PR to remove it.

Reply:
rebased to the latest changes